cmd/internal/opentelemetry/wrapperMux.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opentelemetry
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/cadvisor/version"
+	"go.opentelemetry.io/otel/api/core"
+	"go.opentelemetry.io/otel/api/correlation"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/key"
+	"go.opentelemetry.io/otel/api/propagation"
+	apiTrace "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/exporters/otlp"
+	"go.opentelemetry.io/otel/plugin/othttp"
+	"go.opentelemetry.io/otel/sdk/trace"
+	"google.golang.org/grpc/credentials"
+	"k8s.io/klog/v2"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+var (
+	traceSampler      = flag.String("trace_sampler", "always", "choose a sampler 'never', 'always' or 'probability'")
+	traceSamplerParam = flag.Float64("trace_sampler_param", 0, "used for the 'probability' sampler")
+	traceServiceName  = flag.String("trace_name", "cadvisor", "use localEndpoint service name")
+	traceTags         tracingTags
+)
+
+func init() {
+	flag.Var(&traceTags, "trace_tags", "additional tags for tracing (eg. foo=bar)")
+}
+
+func InitTrace(traceEndpoint, collectorCert string) *otlp.Exporter {
+	exporterOptions := []otlp.ExporterOption{otlp.WithAddress(traceEndpoint)}
+
+	if collectorCert != "" {
+		if creds, err := credentials.NewClientTLSFromFile(collectorCert, ""); err != nil {
```

Comment:
nit: This will be easier to read if we don't if/else when the `if` exits:
change
```golang
if foo, err := bar(); err != nil {
    klog.Fatal(err)
} else {
   // use foo
}
```
to 
```golang
foo, err := bar()
if err != nil {
    klog.Fatal(err)
}
// use foo
```

Reply:
I changed it, I missed the os.Exit on log.Fatal